tests/cases/conformance/classes/members/privateNames/privateNamesUnique-1.ts

```diff
@@ -1,5 +1,6 @@
 // @strict: true
 // @target: es6
+// @strictPropertyInitialization: false

```

Comment:
I like how we're testing this code path as well 👍

Reply:
😅 I have to confess it was more not wanting to change existing tests too much, but it works out I think 😊. The main tests for the option seem good enough and cluttering other tests with extra errors seemed pointless.